include/GraphMatrix.hpp

```diff
@@ -518,6 +521,31 @@ namespace Appledore
             }
         }
 
+        std::vector<EdgeType> bfs(size_t start, size_t maxDepth) {
```

Comment:
Shouldn't the ``start`` variable be of type ``VertexType``? because user is passing in a vertex to start bfs from?

```suggestion
        std::vector<EdgeType> bfs(VertexType start, size_t maxDepth) {
```

Reply:
Yeah, you're rights. I can make that change